package/Dockerfile

```diff
@@ -1,3 +1,7 @@
-FROM alpine
+FROM registry.suse.com/bci/bci-base:15.3
+RUN zypper -n update && \
+    zypper -n clean -a && \
+    rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/* /usr/share/doc/manual/* /var/log/*
```

Comment:
My only concern there would be that zypper's not the only program that uses those directories - while it may be tolerant of these changes other programs may not be. That being said, we are probably ok here for the following reasons:

1) We aren't installing anything new.

2) The only other dir in `/var/log` was `krb5` (kerberos - I don't see any reason why the webhook would need this so if this does cause an issue with kerberos functionality that's probably ok).

@macedogm would you mind adding a comment here (or somewhere adjacent) so that this is documented? I'm thinking something like "Removing potential sparse files created by zypper update to save space"

Reply:
Base image moved to bci-micro. No need to run zypper anymore and we only clean the `zypp` log directory now.

Reply:
The directory is in the image:

```
$ docker run --rm registry.suse.com/bci/bci-micro:15.3 ls -lhaR /var/log
/var/log:
total 12K
drwxr-xr-x  3 root root 4.0K Mar 15 11:35 .
drwxr-xr-x 10 root root 4.0K Mar 22 18:07 ..
drwxr-xr-x  2 root root 4.0K Mar 22 18:07 zypp

/var/log/zypp:
total 20K
drwxr-xr-x 2 root root 4.0K Mar 22 18:07 .
drwxr-xr-x 3 root root 4.0K Mar 15 11:35 ..
-rw-r--r-- 1 root root 9.3K Mar 22 18:07 history
```

I'll file a bug report.